esmvaltool/diag_scripts/aerosols/aero_utils.py

```diff
@@ -0,0 +1,181 @@
+"""Part of the ESMValTool Aerosol diagnostics.
+
+This module contains utility functions commonly used by aerosol
+assessment routines.
+"""
+
+import iris
+import numpy as np
+
+
+class AeroAnsError(Exception):
+
+    """Exception class for errors raised when model data is checked in the
+    extract_pt module.
+    """
+
+
+def add_bounds(cube):
+    """Add bounds to a cubes latitude and longitude coordinates.
+
+    Parameters
+    ----------
+    cube : Iris cube
+        Iris cube with latitude and longitude coordinates.
+
+    Returns
+    -------
+    cube : cube
+        Iris cube with bounds added to the latitude and longitude coordinates.
+    """
+
+    if not cube.coord('latitude').has_bounds():
+        cube.coord('latitude').guess_bounds()
+    if not cube.coord('longitude').has_bounds():
+        cube.coord('longitude').guess_bounds()
+
+    return cube
+
+
+def extract_pt(icube, pt_lat, pt_lon, **kwargs):
+    """Extracts given location(s) (3-D) from a cube.
+
+    Method
+    ------
+    Uses Iris module Analysis.Interpolate to extract values
+    first based on horizontal coordinates and later based on
+    height, if specified.
+
+    If height ('altitude') is requested, checks if cube heights
+    include orography, i.e. HybridHeights have been derived.
+
+    Parameters
+    ----------
+    icube : Iris cube
+    pt_lat, pt_lon : latitude and longitude coordinates of desired points
+    kwargs: (optional)
```

Comment:
```suggestion
    kwargs: (optional, strings)
```

Reply:
or, alternatively, please describe the structural type of each kwarg eg I see `height` could be a list too

Reply:
Done